core/src/main/java/tc/oc/pgm/score/ScoreMatchModule.java

```diff
@@ -60,10 +61,13 @@ public ScoreMatchModule(Match match, ScoreConfig config, Set<ScoreBox> scoreBoxe
     this.config = config;
     this.scoreBoxes = scoreBoxes;
     this.match.getCompetitors().forEach(competitor -> this.scores.put(competitor, 0.0));
-
     if (this.config.mercyLimit > 0) {
       this.mercyRule = new MercyRule(this, config.scoreLimit, config.mercyLimit);
+      scoreLimit = mercyRule.getScoreLimit();
```

Comment:
You cannot do this. Mercy rules change dynamically you cannot save this value, moreover, it just doesn't make sense to save it.

Save a score-override variable, default it to 0 for not-set.